core/server/i18n.js

```diff
@@ -65,12 +65,15 @@ I18n = {
         path = msgPath.split('.');
         path.forEach(function (key) {
             // reassign matching object, or set to an empty string if there is no match
-            matchingString = matchingString[key] || null;
+            matchingString = matchingString[key] || {};
         });
 
-        if (_.isNull(matchingString)) {
+        if (_.isEqual(matchingString, {})) {
             console.error('Unable to find matching path [' + msgPath + '] in locale file.\n');
```

Comment:
Thanks for your PR.

I would suggest to use a proper logging entry here.
`logging.error(new errors.IncorrectUsageError({message: '....'}))`

Both cases can be handled in one condition, because it means the same: You are trying to translate a key, which does not exist.

And furthermore, i think it makes sense to use a fallback translation key instead e.g. `errors.anErrorOccurred`. Otherwise it can happen that we send `i18n error: path "errors.api.requests.invalidFile" was not found.` to the client.

Reply:
Makes sense! Should there be a check if `errors.anErrorOccurred` exists in the specific local? Also, what should the message be? Something like "A translation error occurred"?

Reply:
> should there be a check if errors.anErrorOccurred exists in the specific locale?

hmm not needed for now, because we don't officially support translation the server side keys.